### PR TITLE
v1.1.9 - add support for global variable to use non-production backen…

### PR DESCRIPTION
**Pull request by richard-decastongrene-electro**
…d settings

### CODE REVIEW

File: src/js/ot-dns-script-2.js [hunk 1]
```diff
@@ -8,15 +8,27 @@ let preferences;
 let dnsUI = false;
 
 // Collection Point Information
-const url = 'https://privacyportal.onetrust.com/request/v1/consentreceipts';
-const token =
+var url = 'https://privacyportal.onetrust.com/request/v1/consentreceipts';
```
Comment (electroross) [inline src/js/ot-dns-script-2.js]: Can this be a "let" instead of a var?

**Review by electroross — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -32,3 +32,7 @@
 ## 1.1.8 Updated language
 
 - in DNS popup
+
+## 1.1.9 Support parameter on impoart path to use staging server
+
+ - if window.electroPrivacyStaging set to true on loading, use staging parameters for backend submissions
```
Comment (electroross) [inline CHANGELOG.md]: Can you be more explicit about where this is set, i.e. in DTC's code, with an example.

**Comment by richard-decastongrene-electro** [comment]
Requested changes made

**Review by electroross — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -66,9 +66,11 @@ This code is to simplify the integration of the second OneTrust modal into WordP
 ```
 
 ## UAT Values
-The current production version sends entries to the live OneTrust collection point. If you need to support UAT, you will need to change the following values:
+The current production version sends entries to the live OneTrust collection point by default. If you need to support UAT, set this variable before importing electro-privacy:
+
+window.electroPrivacyStaging = true;
```
Comment (electroross) [inline README.md:71]: Can you add the actual code example so DTC doesn't ask how its done?

Comment (richard-decastongrene-electro) [reply README.md:71]: Not sure what we need here. 

window.electroPrivacyStaging = true;

^^ is the actual code that is needed.

Comment (electroross) [reply README.md:71]: Gotcha. Looks good to me!

**Review by electroross — APPROVED** [review]

**Pull request merged**